com.unity.uiwidgets/Runtime/widgets/texture.cs

```diff
@@ -12,20 +12,20 @@ public class Texture : LeafRenderObjectWidget {
         //         instance.scheduleFrame(false);
         //     }
         // }
-        
-        public Texture(Key key = null, UnityEngine.Texture texture = null) : base(key: key) {
-            D.assert(texture != null);
-            this.texture = texture;
+
+        public Texture(int textureId, Key key = null) : base(key: key) {
```

Comment:
the sequence of parameters are not correct？ should it be (key , textureId)

Reply:
Updated